### DISCUSSION TITLE
Better Mobile Design

### DISCUSSION THREAD

**Discussion by Test1calCutter** [stoatchat/discussions#516] [Feature Suggestions]
Hi
I find the design of Revolt on mobile seems kinda buggy and laggy. This can be fixed/added for sure! 

(you can also add an badge to my profile if you'd like ;) ) 

![7Xu8P0Gm4lpQpPBku7yp2c9Y5Kv92qa391weiNRJTs](https://user-images.githubusercontent.com/108223454/225919294-0751f220-dbc1-46fd-8a7d-3e12ffd16cd3.png)

**Comment by Rexogamer** [comment]
This issue is pretty vague - do you have any specific suggestions?

Also, please don't use issues to ask for badges. If you're waiting for a translator badge, the language you translated (Pig Latin) doesn't have a maintainer; other obtainable badges are listed [here](https://support.revolt.chat/kb/account/badges).

**Reply by Test1calCutter** [reply]
![image](https://user-images.githubusercontent.com/108223454/226129909-2e874de8-4296-49b7-8e5a-51cdaf3d2de8.png)
